Source/scrollrt.cpp

```diff
@@ -315,7 +315,7 @@ static void DrawMonster(const CelOutputBuffer &out, int x, int y, int mx, int my
 	}
 
 	int nCel = monster[m]._mAnimFrame;
-	auto frameTable = reinterpret_cast<const uint32_t *>(monster[m]._mAnimData);
+	auto frameTable = reinterpret_cast<const uint32_t *>(monster[m]._mAnimData->Data());
 	int frames = SDL_SwapLE32(frameTable[0]);

```

Comment:
Unrelated to this PR but it's worth highlighting a bug here -- `Data` is not guaranteed to be 4-byte aligned so this is illegal (e.g. it can cause a crash on MIPS CPUs)

Reply:
This logic is used in multiple places to validate the `CelSprite`.
It would be great if we have method that would perform this validations and also fixes this bug.